### DISCUSSION TITLE
Accidentally deleted Octernship Assignment Repo

### DISCUSSION THREAD

**Discussion by GargiMittal** [community/community#62922] [GitHub Education]
### Hi everyone,

I accidently deleted my  Octernship Assignment Repository is there any way to recover it ? or can I reapply while reapplying it is showing me "There was some issue while accepting the assignment. Please try again later."

**Comment by mysandesh** [comment]
I suggest that you try this: 
https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository

**Comment by mehulrajdev** [comment]
Recover the deleted repository by going to settings on GitHub
